checks.d/mesos_slave.py

```diff
@@ -135,7 +135,7 @@ def _get_constant_attributes(self, url, timeout):
             if state_metrics is not None:
                 self.version = map(int, state_metrics['version'].split('.'))
                 master_state = self._get_state('http://' + state_metrics['master_hostname'] + ':5050', timeout)
-                if master_state is not None:
+                if master_state is not None and 'cluster' in master_state:
                     self.cluster_name = master_state['cluster']
```

Comment:
```python
if master_state is not None and 'cluster' in master_state:
    self.cluster_name = master_state['cluster']
```
could be simplified:
```python
if master_state :
    self.cluster_name = master_state.get('cluster')
```

I am pretty sure we can drop the `if` statement too, and have simply:
```python
 self.cluster_name = master_state.get('cluster') 
```
as `master_state` could not be null at this point.